subprojects/core-api/src/main/java/org/gradle/api/artifacts/repositories/IvyArtifactRepository.java

```diff
@@ -102,15 +102,15 @@ public interface IvyArtifactRepository extends ArtifactRepository, UrlArtifactRe
      * <p>
      * Recognised values are as follows:
      * </p>
-     * <h2>'gradle'</h2>
+     * <h4>'gradle'</h4>
```

Comment:
🤔 Given that we upgrade the JDK version, we should check the generated Javadoc and what it looks like when published. We had some problems with our CSS with the JDK 11 upgrade already.